stock_secondary_unit/views/stock_picking_views.xml

```diff
@@ -23,17 +25,15 @@
                                 '&amp;', ('product_tmpl_id.product_variant_ids', 'in', [product_id]),
                                          ('product_id', '=', False)] or [(0, '=', 1)]"
                     options="{'no_create': True}"
-                    attrs="{'column_invisible': [('parent.immediate_transfer', '=', True)], 'readonly': ['|', ('is_initial_demand_editable', '=', False), '&amp;', '&amp;', ('show_operations', '=', True), ('is_locked', '=', True), ('is_initial_demand_editable', '=', False)]}"
+                    groups="uom.group_uom"
+                    readonly="(not is_initial_demand_editable) or (show_operations and is_locked and not is_initial_demand_editable)"
                     optional="show"
                 />
             </xpath>
-            <xpath
-                expr="//field[@name='move_ids_without_package']/form//field[@name='product_uom_qty']"
-                position="before"
-            >
+            <xpath expr="//field[@name='product_uom_qty']" position="before">
```

Comment:
That xpath is not correct, it duplicates the fields in the tree of the form where you have already added them before.

We have to see how the mobile view is built in this version, if there are changes and if you need to add the fields to make it work exactly the same as the previous version.